proxylib/proxylib/types.go

```diff
@@ -21,12 +21,8 @@ import "C"
 
 // Mirror C types to be able to use them in other Go files and tests.
 
-type OpType uint32
-type OpError uint32
-type Op struct {
-	op      uint32
-	n_bytes uint32 // The number of bytes of data the operation 'op' applies to.
-}
+type OpType int64
+type OpError int64
```

Comment:
Why aren't these `uint64`?

Reply:
See above for `OpType`.

Same applies for `OpError`, but additionally it consumes the same space as `n_bytes` does for non-error operations.